test/wdio/package.json

```diff
@@ -11,10 +11,10 @@
     "@stencil/core": "file:../..",
     "@stencil/sass": "^3.0.10",
     "@types/node": "^16.11.7",
-    "@wdio/browser-runner": "^8.33.1",
-    "@wdio/cli": "^8.33.1",
-    "@wdio/globals": "^8.33.1",
-    "@wdio/mocha-framework": "^8.33.1",
+    "@wdio/browser-runner": "^8.35.1",
```

Comment:
Are the `@wdio` upgrades strictly needed for this PR? If possible, I'd prefer we let Renovatebot handle these - that way, we can separate test implementations from dependency upgrades, which makes bisecting/debugging commits a bit easier (especially as we're all swarming on this area of the codebase right now)

Reply:
yes, there are some bugs I fixed in WebdriverIO that would break the tests without the update. Alternatively we can merge https://github.com/ionic-team/stencil/pull/5498 and I can just rebase and move forward without dep updates (as the other PR includes them)